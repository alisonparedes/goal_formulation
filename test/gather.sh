python ../src/agent.py ../test/gather_10x1_0_real.world ../test/gather_10x1_0_belief.world 20 10 2 10 1 0






