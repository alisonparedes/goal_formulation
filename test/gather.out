reality: ../test/gather_10x1_0_real.world
belief: ../test/gather_10x1_0_belief.world
horizon: 20
sample: 10
max_food: 2
time: 10
seed: 1
known: 0


time: 0
new reward: 0
tot reward: 0
-b--F--F-- ?b?????F??

time: 1
new reward: -1
tot reward: -1
-BH-F--F-- ?BH????F??

time: 2
new reward: -1
tot reward: -2
-B-HF--F-- ?B-H???F??

time: 3
new reward: -1
tot reward: -3
-B--$--F-F ?B??$??F??

time: 4
new reward: -1
tot reward: -4
-B-$---F-F ?B?$-??F??

time: 5
new reward: -1
tot reward: -5
-B$----F-F ?B$--??F??

time: 6
new reward: 49
tot reward: 44
-b-----F-F ?b---??F??

time: 7
new reward: -1
tot reward: 43
-BH----F-F ?BH--??F??

time: 8
new reward: -1
tot reward: 42
-B-H---F-F ?B-H-??F??

time: 9
new reward: -1
tot reward: 41
-B--H--F-F ?B--H??F??

time: 10
new reward: -1
tot reward: 40
-B---H-F-F ?B---H?F??

summary: 40 ../test/gather_10x1_0_belief.world

